### PR TITLE
toggle buttons for reminder text

### PR DESCRIPTION
**Pull request by sghiaa**
#### Summary of changes
**Asana Ticket:** [📈 Add explanation for prediction-accuracy metric calculation](https://app.asana.com/0/584764604969369/1118056447625835)

heres an attempt at css toggling that seems ok to me.  open to doing it differently though.

#### Reviewer Checklist
- [ ] Meets ticket's acceptance criteria
- [ ] Any new or changed functions have typespecs
- [ ] Tests were added for any new functionality (don't just rely on Codecov)

### CODE REVIEW

File: lib/prediction_analyzer_web/templates/accuracy/index.html.eex [hunk 1]
```diff
@@ -20,8 +20,70 @@
...
+          <a href="#tab2" class="nav-tab nav-tab-active reminder-text-hidden">Hide</a>
+        </p>
+        <p>
+          A snapshot of all predictions is taken once per minute. Each prediction is matched to its corresponding
+          actual arrival or departure time based on the vehicle ID and GTFS trip ID, and passes or fails the
+          accuracy standard based on the below bins. If no match is found for the vehicle ID and GTFS trip ID, the
+          prediction is recorded as failing.
+        </p>
+
+        <table>
```
Comment (losvedir) [inline lib/prediction_analyzer_web/templates/accuracy/index.html.eex]: `class="table"` will help as that will pull in the bootstrap styling.

Comment (sghiaa) [reply lib/prediction_analyzer_web/templates/accuracy/index.html.eex]: sounds good

File: assets/css/app.css [hunk 2]
```diff
@@ -111,3 +111,6 @@
 .sort-order-link {
   float: right;
 }
+
+#reminder-text .reminder-text{display:none;}
```
Comment (losvedir) [inline assets/css/app.css]: I believe generally styling with IDs is frowned upon. I think these rules can be just `.reminder-text`, similar to the rest of the selectors.

Comment (sghiaa) [reply assets/css/app.css]: ah, yeah i had been trying to reduce down from https://jsfiddle.net/hoq0djwc/1/ but i didnt think to remove the id selector as well.

**Pull request merged**